packages/wallet/src/contract-tests/transactions.test.ts

```diff
@@ -165,7 +165,7 @@ describe("transactions", () => {
       channel,
       allocation: ["0x05", "0x05"],
       destination: [participantA.address, participantB.address],
-      turnNum: 6,
+      turnNum: 8,
```

Comment:
I've noticed this in my work on #143 and #152 that these numbers need to be exactly right across these test files and utils. It would be worth spending a bit of time to make selecting them idiot-proof.